Source/SwiftLintFramework/Models/Linter.swift

```diff
@@ -137,4 +137,35 @@ public struct Linter {
         }
         return corrections
     }
+
+    public func gitPatchCorrect() -> String? {
+        guard let fileCopy = file.temporaryCopy() else { return nil }
+        for rule in rules.flatMap({ $0 as? CorrectableRule }) {
+            let newCorrections = rule.correct(file: fileCopy)
+            if !newCorrections.isEmpty {
+                fileCopy.invalidateCache()
+            }
+        }
+
+        // Generate git patch to apply corrections
+        let task = Process()
+        task.launchPath = "/usr/bin/git"
```

Comment:
I'm using latest git by Homebrew that installed at `/usr/local/bin/git`.
It would be better that respecting `PATH`:
```swift
task.launchPath = "/usr/bin/env"
task.arguments = ["git", "diff", "--no-index", file.path!, fileCopy.path!]
```